frontend/src/utilsLogic.tsx

```diff
@@ -173,18 +173,44 @@ export const updateQueryOwnedInstancesQuery = (
     }
 
     !isPrettyNameUpdate &&
-      notifyStatus(
-        instance!.status?.phase!,
-        instance!,
-        updateType!,
-        tenantNamespace,
-        WorkspaceRole.user
-      );
+      notifyStatus(instance!.status?.phase!, instance!, updateType!);
 
     return prev;
   };
 };
 
+export enum subObjType {
+  Deletion,
+  UpdatedInfo,
+  PrettyName,
+  Addition,
+  Drop,
+}
+
+export const getSubObjType = (
+  oldObj: ItPolitoCrownlabsV1alpha2Instance,
+  newObj: ItPolitoCrownlabsV1alpha2Instance,
+  uType: UpdateType
+) => {
+  if (uType !== UpdateType.Deleted) {
+    const { spec: o_spec, status: o_status } = oldObj;
```

Comment:
Maybe it's better to stick with camelCase. oldSpec, oldStatus and alike should be fine here.